Packs/QualysVulnerabilityManagement/Integrations/QualysVulnerabilityManagement/QualysVulnerabilityManagement.py

```diff
@@ -0,0 +1,1499 @@
+            generated from the parsed response
+    """
+    command_parse_and_output_data = kwargs['command_parse_and_output_data']
+    handled_result = kwargs['handled_result']
+    asset_collection = handled_result[command_parse_and_output_data['collection_name']]
+
+    headers = command_parse_and_output_data.get('table_headers') if\
+        command_parse_and_output_data.get('table_headers') else None
+    parsed_output = generate_list_dicts(asset_collection)
+    readable_output = tableToMarkdown(
+        name=command_parse_and_output_data['table_name'],
+        t=parsed_output,
+        headers=headers
+    )
+    return parsed_output, readable_output
+
+
+def build_changed_names_output(**kwargs) -> Tuple[List[Any], str]:
+    """
+    Takes the output and changes the output fields names as described in the command data
+    Args:
+        **kwargs:
+                handled_result (Dict): response received from Qualys API
+                command_parse_and_output_data (Dict): Data for parsing and creating output for a specific command
+    Returns:
+            Tuple containing a dictionary and a markdown table generated from the response
+    """
+    command_parse_and_output_data = kwargs['command_parse_and_output_data']
+    output = kwargs['handled_result']
+
+    output = change_list_dicts_names(command_parse_and_output_data, output)
+    readable_output = tableToMarkdown(name=command_parse_and_output_data['table_name'], t=output)
+
+    return output, readable_output
+
+
+def build_multiple_text_options_output(**kwargs) -> Tuple[None, str]:
+    """
+    When there's no need to build output from the response but output text is based on command's action requested
+    this function will take the text based on the action and will return it
+    Args:
+        **kwargs:
+                command_parse_and_output_data (Dict): Data for parsing and creating output for a specific command
+    Returns:
+            Tuple containing None and text based on the action requested
+    """
+    command_parse_and_output_data = kwargs['command_parse_and_output_data']
+    action = args_values['action']
+
+    readable_output = command_parse_and_output_data['output_texts'][action]
+
+    return None, readable_output
+
+
+def build_text_value_pairs_parsed_output(**kwargs) -> Tuple[Dict[str, Any], str]:
+    """
+    A command might have multiple key value pairs, where the name of the key is an attribute of <KEY> element
+    this builder will create an output of those keys
+    Args:
+        **kwargs:
+                handled_result (Dict): response received from Qualys API
+                command_parse_and_output_data (Dict): Data for parsing and creating output for a specific command
+    Returns:
+        Tuple containing a dictionary created from those key-value pairs and a markdown table as a string
+    """
+    command_parse_and_output_data = kwargs['command_parse_and_output_data']
+    output = kwargs['handled_result']
```

Comment:
The usage of kwargs throughout my code helps me handle the difference in structure of output of commands. I store the output builder of each command in a dictionary and then pass the relevant key from the dictionary, which holds a callable. Because each callable builds output in a different way I use kwargs so I can pass the relevant arguments each output builder needs.

The same process also happens with the handlers. If I add the arguments as you suggested it means that many of the output builders will have arguments which they never use.